config/gregtech/worldgen/overworld/olivine_vein2.json

```diff
@@ -1,39 +1,44 @@
 {
-  "weight": 55,
-  "density": 0.3,
-  "max_height": 50,
-  "min_height": 10,
-
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [13, 13]
-  },
-  "filler": {
-    "type": "simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 5,
-          "value": "ore:prasiolite"
-        },
-        {
-          "weight": 30,
-          "value": "ore:bentonite"
-        },
-        {
-          "weight": 30,
-          "value": "ore:magnesite"
-        },
-        {
-          "weight": 30,
-          "value": "ore:olivine"
-        },
-        {
-          "weight": 10,
-          "value": "ore:glauconite"
+    "weight": 55,
+    "density": 0.3,
+    "max_height": 50,
+    "min_height": 10,
+    "generator": {
+        "type": "ellipsoid",
+        "radius": [
+            13,
+            13
+        ]
+    },
+    "filler": {
+        "type": "simple",
+        "value": {
+            "type": "weight_random",
+            "values": [
+                {
+                    "weight": 5,
+                    "value": "ore:prasiolite"
+                },
+                {
+                    "weight": 30,
+                    "value": "ore:bentonite"
+                },
+                {
+                    "weight": 30,
+                    "value": "ore:magnesite"
```

Comment:
Other than magnesite, this vein is extremely useless. Prasiolite is only used in magneto resonatic dust, used only very late game.

Reply:
also still generates, but cant be crafted